gluoncv/model_zoo/model_zoo.py

```diff
@@ -406,6 +406,9 @@ def get_model(name, **kwargs):
         err_str = '"%s" is not among the following model list:\n\t' % (name)
         err_str += '%s' % ('\n\t'.join(sorted(_models.keys())))
         raise ValueError(err_str)
+    num_classes = kwargs['classes']
```

Comment:
@sallypannn not sure why you modify get_model function here? can you explain?

Reply:
Now this part is moved to ImageClassificationEstimator. @zhreshold